package.json

```diff
@@ -331,11 +331,12 @@
     "esbuild": "^0.13.15",
     "eslint": "^7.26.0",
     "mocha": "^10.0.0",
-    "prettier": "^2.7.1",
     "path-browserify": "^1.0.1",
+    "prettier": "^2.7.1",
     "ts-loader": "^9.2.3",
     "typescript": "^4.4.3",
     "webpack": "^5.44.0",
-    "webpack-cli": "^4.7.2"
+    "webpack-cli": "^4.7.2",
+    "media-typer": "^1.1.0"
```

Comment:
it looks like being used in source code so better not be in devDependencies. Suggest to be in the dependencies entry in client/package.json. It won't cause real issues as it will be bundled and not publish npm modules. But just make it clearer. client and server can be separated when needed in future, so their dependencies are in their own package.json file.